src/placesmodel.cpp

```diff
@@ -429,6 +429,18 @@ void PlacesModel::onMountRemoved(GVolumeMonitor* monitor, GMount* mount, PlacesM
 }
 
 void PlacesModel::onVolumeAdded(GVolumeMonitor* /*monitor*/, GVolume* volume, PlacesModel* pThis) {
+    // the item may have been added with "mount-added" (as in loopback mounting)
+    bool itemExists = false;
+    GMount* mount = g_volume_get_mount(volume);
```

Comment:
How about using the `GObjectPtr<GMount>`...?

Reply:
Equivalent and no real gain.

Reply:
> Equivalent and no real gain.

Try to rewrite the all the libfm-qt w/o using `GObjectPtr` ...

Reply:
?!
`GObjectPtr` wasn't used in that file, so I kept the style. At some point, all such cases should be found and changed but in a coherent way.

Reply:
A simple way of doing that could be searching for `g_object_unref`. Something for another PR....